### PR TITLE
test: Adjust for fedora 32

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

**Comment by marusak** [comment]
It got stuck, but I have some reasonable [results](https://logs.cockpit-project.org/logs/pull-13603-20200224-174931-5bfb6451-fedora-32-bots-568/log.html) (waiting until it finishes to see some screenshots)

1. `test/verify/check-journal TestJournal.testAbrt*` fail. Maybe the message changed?
2. Bunch of storage tests failed. It prints `Failed to get md node for array 'ARR': Failed to read the symbolic link /dev/md/ARR: No such file or directory` which might be related.
3. Bunch of storage tests failed. It prints `Empty reply from server` which might be related.
4. Bunch of machines tests failed. It prints `GetDomainCapabilities failed: %s` which might be related.
5. Firewall seems to have a new AVC 
```
audit: type=1400 audit(1582569630.930:313): avc:  denied  { sys_nice } for  pid=631 comm="firewalld" capability=23  scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:system_r:firewalld_t:s0 tclass=capability permissive=0
```

**Comment by marusak** [comment]
'AddDisk` is still broken [log](https://logs.cockpit-project.org/logs/pull-13603-20200224-174931-5bfb6451-fedora-32-bots-568/log.html#230-2) but I am re-running tests to get new results. Most of the failures should be hopefully fixed/skipped.

**Comment by marusak** [comment]
[Second run](https://logs.cockpit-project.org/logs/pull-13603-20200225-134316-9e905701-fedora-32-bots-568/log.html) went much better.
I am pushing some more changes, but there are still two tests, that do not work (both machines):
1. `testConfigureBeforeInstall` - It changes boot order. By default only `disk` is selected, it also adds `network` and moves it into the first place. It saves it and then installs the VM. After the installation the `network` disappears from `Boot Order` and only disk is kept. This may be related to my last commit in this PR - the script was passing strings `"\n"` to `--graphics` and to `--network` which made the whole `pkg/machines/scripts/install_machine.sh` script fail (check for empty string failed, thus it parsed it as `--graphics <value_of_the_string>` but there was no value so `virt-install` failed)
2. `testCreate` - It checks in one VM for list of disk and checks that they have device `disk` but the device is `cdrom`.

**Comment by marusak** [comment]
Now it [looks](https://logs.cockpit-project.org/logs/pull-13603-20200226-105638-b487ad8f-fedora-32-bots-568/log.html) great :) `testCreate` is stuck, but now I know why - it does `reload()` and chrome opens popup asking if we want to reload the page. No idea why it only appeared now.

File: test/verify/check-machines-dbus [hunk 1]
```diff
@@ -298,11 +298,22 @@ class TestMachinesDBus(machineslib.TestMachines):
...
+                m.execute("systemctl restart libvirtd")
+                b.click("tbody tr[data-row-id=vm-VmNotInstalled] th")
+
             b.mouse("#vm-VmNotInstalled-firmware-tooltip", "mouseover")
             b.wait_in_text("#missing-uefi-images .tooltip-inner", "Libvirt did not detect any UEFI/OVMF firmware image installed on the host")
             b.mouse("#vm-VmNotInstalled-firmware-tooltip", "mouseout")
             b.wait_not_present("#missing-uefi-images")
             m.execute("mv /tmp/ovmf/* {0}".format(ovmf_path))
+
+            if m.image == "fedora-32":
```
Comment (martinpitt) [inline test/verify/check-machines-dbus]: Would be nice to add the `# HACK` comment again here, so that it appears in grepping/searching.

**Pull request merged**